utopian-io/v2.utopian.io
# [FEATURE] Google & Linkedin Sign in

### My PR doesn't concern a Jira task
Implements Linkedin sign in:

- Sign in with a Linkedin account
- Create account from Linkedin sign in
- Cross sign in between auth providers
---
## New env vars:

### packages/auth
* LINKEDIN_CLIENT_ID=''

### packages/api
* LINKEDIN_CLIENT_ID=''
* LINKEDIN_CLIENT_SECRET=''
* LINKEDIN_REDIRECT_URL=http://localhost:8081

### **Test plan**

_Sign up with Linkedin_ 

* Delete your user account or test with a blank database
* Go to localhost:8080 and click on Sign In
* In the auth app, click on Sign in with Linkedin
* Accept the request and you'll be redirected to the signup page
* Create a username 
* You should be redirected to the main app

_Sign in with Linkedin_ 
* After creating your account, log out
* Click on Sign In again
* Click on Sign In with Linkedin
* You should be redirected to the main app

_Sign in with Github account with same email_
* Make sure your github account has the same email that your Linkedin account
* Sign out and sign in with Github
* You should be redirected to the main app (not to the sign up page)
